crates/simdutf/src/generated.rs

```diff
@@ -1123,3 +1123,35 @@ pub unsafe fn convert_valid_utf32_to_utf16be(src: *const u32, len: usize, dst: *
 pub unsafe fn convert_valid_utf32_to_utf16le(src: *const u32, len: usize, dst: *mut u16) -> usize {
     crate::bindings::simdutf_convert_valid_utf32_to_utf16le(src, len, dst)
 }
+
+/// Convert base64 string into binary data.
+///
+/// # Safety
+/// + `src` and `dst` must be non-null and properly aligned.
```

Comment:
The argument names mismatches. 
You can add a new documentation generator function which supports specifying the names.

Reply:
Updated to fix this